### PR TITLE
Motion comments inline editing

### PR DESCRIPTION
**Pull request by FinnStutzenstein**
@emanuelschuetze @normanjaeckel Whats about making a generic inline editing factory? For this PR I copied 95% of the `MotionTextInlineEditing` to `MotionCommentInlineEditing` and just changed the specific saving the the factory.

### CODE REVIEW

**Comment by FinnStutzenstein** [comment]
@emanuelschuetze please retest.

**Comment by FinnStutzenstein** [comment]
@normanjaeckel Done.

File: openslides/motions/static/js/motions/site.js [hunk 1]
```diff
@@ -1139,6 +1139,51 @@ angular.module('OpenSlidesApp.motions.site', [
...
+                function (obj) {
+                    return motion['comment ' + field.name];
+                },
+                function (obj) {
+                    motion['comment ' + field.name] = obj.editor.getData();
+                }
+            );
+            $scope.commentsInlineEditing.push(inlineEditing);
+        });
+        $scope.commentsInlineEditingSaveToolbarVisible = function () {
```
Comment (normanjaeckel) [inline openslides/motions/static/js/motions/site.js]: Can we please find a solution not to put these stuff into the "global" scope of this controller? The `$scope.commentsInlineEditing` can be a container that hold all general control functions (like this one) and all inlineEditing instances which contains themselves control functions for single comment fields (like save, revert, enable, disable, ... I think).

File: openslides/motions/static/templates/motions/motion-detail.html [hunk 2]
```diff
@@ -452,15 +452,45 @@ <h3 translate>Version history</h3>
...
-  <h3 translate>Comments</h3>
-  <div ng-repeat="field in commentsFields">
-    <div ng-if="(field.public && !field.forState && !field.forRecommendation) ||
-        (operator.hasPerms('motions.can_see_and_manage_comments') && !field.forState && !field.forRecommendation)">
-      <h4>
-        {{ field.name }}
-        <span ng-if="!field.public" class="label label-warning" translate>internal</span>
-      </h4>
-      <div ng-bind-html="motion.comments[$index]"></div>
+  <div class="row">
```
Comment (normanjaeckel) [inline openslides/motions/static/templates/motions/motion-detail.html]: Can you put this stuff into an extra file and use ng-include? Like we do for normal text editing and toolbar.

**Comment by FinnStutzenstein** [comment]
@normanjaeckel Ok, all done. Now the complete comments container is separated in a `comments.html`.

**Comment by normanjaeckel** [comment]
Ah. Please don't forget to add a CHANGELOG entry.

**Comment by FinnStutzenstein** [comment]
@normanjaeckel Ok, now there is a new factory for comments inline editing. I didn't want to change the `MotionInlineEditing` because this is as generic as posssible to create **one** instance.

**Comment by normanjaeckel** [comment]
Ok. Rebase.

**Comment by FinnStutzenstein** [comment]
@normanjaeckel Rebased.

**Pull request merged**